docs/references/files/fossa-yml.md

```diff
@@ -127,6 +130,11 @@ The `name:` and `release:` of the release group's release to add your project to
 
 If you choose to associate a project with a release group, you **must** supply both name and release.
 
+#### `project.labels:`
+The `labels` field allows you to add labels to projects so that you can classify certain projects how you would like. This adds a more flexible way to query for projects as opposed to assigning a project to a team.
+
+Up to 5 labels are allowed to be associated with a project at a time.
```

Comment:
If I scan the project with one label, then scan it later with two different labels do the new project labels get added or replace the old ones? If it's additive, what happens if I add something above 5?

Since this behavior is probably controlled by core, I can see this being more appropriately documented elsewhere. If it is, maybe include a link?

Reply:
This isn't publicly documented yet, but I added information about what happens if 5 labels already exist. Thats a good thought, thanks.